src/containers/network/NetworkEditor.js

```diff
@@ -165,19 +165,20 @@ export default class NetworkEditor extends React.Component {
     dispatch(typeahead(term, typeaheadId, {type: 'communities', moderated: true}))
   }
 
-  getCommunityChoices = term => {
-    if (!term) {
-      return []
-    }
-
-    let { networkEdit: { communities } } = this.props
-    const { currentUser } = this.context
-    var match = c =>
-      startsWith(c.name.toLowerCase(), term.toLowerCase()) &&
-      !contains(communities, c.id)
-
-    return filter(filter(currentUser.memberships, m => m.role === 1).map(m => m.community), match)
-  }
+  // LEJ: Moved into Redux action and reducer and no longer used
```

Comment:
let's keep things tidy in the codebase; no need to keep commented-out stuff around, since it's in the git history if we ever need to retrieve it.